src/OTSession.js

```diff
@@ -41,6 +41,7 @@ export default class OTSession extends Component {
       apiKey: this.props.apiKey,
       sessionId: this.props.sessionId,
       token: this.props.token,
+      options: this.props.options,
```

Comment:
Since there is no default value, this is causing the tests to fail. Could you please address this?